app/components/grades/gradeInfo.js

```diff
@@ -48,67 +41,52 @@ const getGradeInfo = (student, gradeData) => {
             date:   exams[i].date
         };
 
-        const assembledInfo = assembleGradeInformation(
-            studentId,
-            gradeData,
-            examData,
-            grades
-        );
+        const assembledInfo = assembleGradeInformation(studentId, gradeData, examData, grades);
         grade.push(assembledInfo);
     }
 
     return grade;
-};
+}
 
-const gradeAvgDenominator = grades => _.sumBy(
-    _.filter(grades, grade => grade.score > 0),
-    grade => parseInt(grade.weight, 10)
-);
+function gradeAvgDenominator(grades) {
+    return _.sumBy(_.filter(grades, grade => grade.score > 0),
```

Comment:
```suggestion
    return _.sumBy(_.filter(grades, 'score'),
```

this should work because `0` is a falsy value

Reply:
fixed